### ISSUE TITLE
is this dead?

### ISSUE THREAD

**Issue by r3k2** [gopwn/pwn#11]
Hello, I just saw this project and I am a heave user of pwntools, but will love to switch from python to go as much as possible for diff reasons, are you guys going to continue to develop for this project?

**Comment by Matt146** [comment]
Yup, it's pretty much dead. I may continue development on my own later if I find the time and motivation to do so

**Comment by r3k2** [comment]
@Matt146 anyway we could help? some people we are so tired of python BS, will be nice to have this type of tool be faster and lower level C/GO etc. maybe instead of rewriting pwntools we could write a radare2 plugin in GO that does similar things than pwntools or peda for GDB.

**Comment by Matt146** [comment]
@r3k2 First and foremost, I am truly sorry for the late response. I was planning on creating my own version of gopwn, where I focus on developing something similar to metasploit in go, as I am not sure if the other contributors of the project are still interested. However, I am currently swamped with work but I would be interested in continuing development on a new version of gopwn later. With that said, if you would like to, feel free to make a pull request and I'll personally take a look at it.

**Comment by r3k2** [comment]
@Matt146 I am very interested, so let me know.

**Comment by UlisseMini** [comment]
Just saw this issue, sorry for abandoning this! I don't write much go anymore but if anyone wants to fork & maintain this that would be great :)

**Comment by Jacute** [comment]
@r3k2 @UlisseMini I started writing a similar library. Would be great if someone is interested and joins the development
https://github.com/Jacute/gowntools
